hack/dockerfile/install/rootlesskit.installer

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
 
 # When updating, also update rootlesskit commit in vendor.conf accordingly
-# v0.14.6
-: "${ROOTLESSKIT_VERSION:=b25a0bad15d664c4ee8885ba622569425e918a68}"
+# v1.0.0
+: "${ROOTLESSKIT_VERSION:=1920341cd41e047834a21007424162a2dc946315}"
```

Comment:
Wondering if we should continue to use a commit for this one, or also switch to use the version, as this is used as (e.g.)

```bash
go install .... github.com/rootless-containers/rootlesskit/cmd/.../@${ROOTLESSKIT_VERSION}`
```

We also need to change the `vendor.conf` to `vendor.mod` in the comment 😂

Reply:
I suggest to continue using the hash, to minimize the risk of supply chain attacks.

Reply:
Yeah, it's not urgent, just that we'd already have the same in `vendor.mod` / `go.mod`